### ISSUE TITLE
Hovering only possible on current "active" file

### ISSUE THREAD

**Issue by maziac** [microsoft/debug-adapter-protocol#86]
Hi,

don't know if this is intended behavior or a bug. If it's intended then please turn this into a feature request.

I'm using the evaluateRequest to be informed about the cursor hovering about some string in the editor.
This is working if the file is the current file being debugged. I.e. the one where the program counter is.
Hovering in other source code files doesn't show any effect. evaluateRequest isn't even called.

**Comment by puremourning** [comment]
In what was is that related to the protocol? I don’t see any such restriction in the protocol (though my client also behaves this way so I’d be curious to understand your use case)

**Comment by maziac** [comment]
Use case:
debug in file1.
Switch to file2.
Hover over a variable in file 2.
Nothing happens.
Why should I be not able to view e.g. global variables or constants in file2?

**Comment by puremourning** [comment]
Agreed. But the first point is the key - it’s your client making this decision, not the _protocol_

**Comment by maziac** [comment]
No, it's not. Or maybe I'm missing something.
The evaluateRequest is called from the debug adapter protocol.
But it simply isn't called. (Only for the debugged file.)
